zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js

```diff
@@ -209,7 +209,7 @@ const styles = theme => ({
     backgroundColor: '#dc3545',
     boxSizing: 'border-box',
     width: '100%',
-    textAlign: 'left',
+    textAlign: 'center',
```

Comment:
This will affect footer for full-width non-mobile cases also and will mess up the UX. Before approaching the PR, can we continue discussion on the related issue to see if this is really needed?

Reply:
Hey! @tuxology As you can see in non-mobile cases its looking fine only
![after-2(1)](https://user-images.githubusercontent.com/67657066/164626447-fab4bf03-40a5-436e-8bb5-0873cdaeaa2f.png)
Before that the alignment was not in the centre

![WhatsApp Image 2022-04-22 at 7 26 20 AM](https://user-images.githubusercontent.com/67657066/164627402-8beded11-744d-4281-8fce-6e1fe8a0d25e.jpeg)